Add: Gravity behavior and JetPack

xxfast opened the pull request:
now the player gravitate towards the center whilst not moving
![iy6wh9oxfdl1e](https://cloud.githubusercontent.com/assets/13775137/25582513/0ef822a2-2ed1-11e7-9203-60837b96ad23.gif)

xxfast committed:
added the gravity behaviour

the player is attracted towards the centre object

xxfast committed:
added the jetpack and gravity behaviour

now the player gravitate towards the center whilst not moving

xxfast commented:
![](https://github.trello.services/images/mini-trello-icon.png) [Player Mechanic - Gravity behavior](https://trello.com/c/Xvuqld6E/12-player-mechanic-gravity-behavior)

xxfast committed:
Merge branch 'master' into gravity-behavior

xxfast committed:
Merge branch 'master' into gravity-behavior

JPGalovic reviewed (changes requested):
1. Particle should be only displayed while player is actully flying (only if both "Should Gravitate" from Gravity Behavior and "Moving" from stamina are both true)

2. Behavior should be added to player prefabs (all of them) in the resources not just the scene, as behavior should be editable through Resources Folder directly rather than on a scene by scene basis.

xxfast committed:
Added the prefab with jetpack

didn’t add the jet pack to other prefabs, simply because this mechanic
is unique to one theme

xxfast committed:
updated so that the particles go away while not moving

xxfast committed:
Merge branch 'master' into gravity-behavior

JPGalovic reviewed (approved):
Works Well
